src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -500,7 +500,7 @@ await foreach (var item in enumerable)
             [EnumeratorCancellation] CancellationToken cancellationToken = default)
         {
             var enumerable = ReceiveWhileAsync(filter, max, idle, msgs, cancellationToken)
-                .ConfigureAwait(false).WithCancellation(cancellationToken);
+                .WithCancellation(cancellationToken);
```

Comment:
We don't need `.WithCancellation()` here, token is already passed into the method.

Reply:
That call was there previously and I think I just kept it during search and replace, but yes I'll clean that up.